_posts/2016-09-16-angular-2-animation-important-concepts

```diff
@@ -0,0 +1,236 @@
+---
+layout:     post
+title:      "Angular 2 Animations - Foundation Concepts"
+
+date: 2016-06-08
+imageUrl: '/images/banner/angular-2-component-animations.jpg'
+
+summary: "Animation in Angular 2 is now easy and more intuitive... Learn foundational animation concepts and start animating your Angular 2 components!"
+
+categories:
+  - angular
+
+tags:
+  - angular2
+  - animation
+  - components
+  - relative paths
+
+topic: components
+
+author: thomas_burleson
+---
+
+
+### Our Scenario
+
```

Comment:
How about a little intro paragraph that prepares the reader what to expect in this article?